core/controllers/get_filters.py

```diff
@@ -66,19 +66,27 @@ def get_outcome_categories():
     return outcome_categories, 200
 
 
-def get_regions():
-    """Returns all unique itl region codes associated with projects.
+def get_geospatial_regions():
+    """Returns all unique ITL1 region codes associated with projects.
 
-    :return: a list of itl region codes
+    :return: a list of ITL1 region codes and region names in dictionaries
```

Comment:
I know this might be pedantic, but could you update the docstring to match https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/c17a8c7680f9a1469b94ea49ffb0337d8ef583ab/core/controllers/get_filters.py#L37 as really it does slightly more than just return a list of dicts.